projects/plugins/jetpack/modules/widget-visibility/widget-conditions.php

```diff
@@ -712,6 +712,10 @@ public static function widget_update( $instance, $new_instance, $old_instance )
 	public static function sidebars_widgets( $widget_areas ) {
 		$settings = array();
 
+		if ( ! is_array( $widget_areas ) ) {
+			return;
```

Comment:
I would return `$widget_areas` to avoid damaging things further. I assume that another plugin filtered `sidebars_widgets` and returned the wrong thing, hence the notice, but I think we should still return what was given to us, to avoid causing more issues by getting rid of `$widget_areas` entirely.

```suggestion
			return $widget_areas;
```